libs/cv_engine.py

```diff
@@ -108,3 +146,28 @@ def process_video(self, video_uri):
 
     def stop_process_video(self):
         self.running_video = False
+
+    def reset_log_detail(self):
+        self.log_detail = {}
+        for section in LOG_SECTIONS:
+            if section == "Post processing steps":
```

Comment:
Can we put this string in a constant at the top of this file?

Reply:
Done